simple.hs

```diff
@@ -0,0 +1,15 @@
+module Main where
+
+import Data.Char (toLower)
+import Data.List (sortBy)
+import qualified Data.Map.Strict as M
+
+main :: IO ()
+main = (putStr . prettify . sortCount . wc) =<< getContents
+  where
+    sortCount = sortBy (\(_, a) (_, b) -> compare b a) . M.toList
+    prettify = unlines . fmap (\(w, c) -> w ++ " " ++ show c)
+
+wc :: String -> M.Map String Int
+wc = foldr insertWord M.empty . words
```

Comment:
Why not `M.fromListWith (+) . flip zip [1,1..] . map toLower . words`? fromList is more idiomatic and should be faster than foldr

Reply:
Ok, I got you. =) I'll send a PR to your repo now